float32 x
float32 y
float32 theta_deg 
---
bool status
